scriptworker/constants.py

```diff
@@ -204,73 +204,77 @@
     }), ),
```

Comment:
Nit: We also need to key `valid_vcs_rules`.

Reply:
Sure, this will allow keying any top-level config entry, but I think adding keying can wait for a separate PR where other keys are actually added.